language.md

```diff
@@ -109,4 +111,4 @@ If you speak Urdu [click here](urdu/urdu-news.md)
 
 Ako govorite Bosanski jezik/If You speak Bosnian language [kliknite ovdje/click here](Bosnian/Bosnian_story.md)
 
-Nếu bạn nói tiếng Việt, [nhấn vào link này](vietnamese/kechuyen.md).
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.